### ISSUE TITLE
PayPal Payments Gateway Not Showing In Checkout for Subscription Order

### ISSUE THREAD

**Issue by MachineITSvcs** [woocommerce/woocommerce-paypal-payments#1617]
### Describe the Bug

From the PayPal Payments documentation online, my understanding was that PayPal Payments supported subscriptions using WooCommerce Subscriptions. I have WooCommerce Subscriptions installed, but I do not see the PayPal Payments gateway during checkout of a subscription product. In the meantime I've re-enabled WooCommerce PayPal Checkout Gateway as that still seems to work correctly for subscriptions. Maybe I misunderstood the online documentation?

### Expected Behavior

I would expect the PayPal gateway to show during checkout so it could actually be used.

### Actual Behavior

Gateway is not showing in checkout for subscription product/order.

### Environment

 - WordPress Version: 6.3
 - WooCommerce Version: 8.0.2
 - WooCommerce Subscriptions Version: 5.4.0
 - Plugin Version: 2.2.0
 - Browser: Chrome

**Comment by InpsydeNiklas** [comment]
Hi @MachineITSvcs 
It seems that your GitHub issue is primarily a support request. While we value your input and feedback, GitHub is predominantly used for our development activities. For any questions regarding the plugin setup, we kindly ask that you reach out directly to our [support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help).

The requirements to use PayPal Payments with subscriptions are listed in the documentation here: https://woocommerce.com/document/woocommerce-paypal-payments/#subscriptions-faq
In short, your PayPal account must be enabled for Reference Transactions, and the Vaulting feature must be activated for the plugin to declare subscription support.

We'll close this GitHub issue for now, but the [support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help) is ready to assist you with any remaining questions. Thank you for your understanding!

**Comment by MachineITSvcs** [comment]
I reported it here because it appears to be a bug. The PayPal Standard
plugin, and PayPal support, both report Reference Transactions are enabled,
but the PayPal Payments plugin does not. I see a bunch of 403 UNAUTHORISED
ACCESS/Invalid Scopes to the billing agreement endpoint, which according to
the request description is to check if Reference Transactions are enabled.
I have put in a merchant ticket with PayPal as well in case this is
something on their end as I was approved for Reference Transactions back in
2018 and haven't had any issues, but the other day I applied and was
approved for Advanced Debit and Credit Card Payments, and I wanted to make
sure doing this didn't somehow disable Reference Transactions on my
account. PayPal support claimed it was still enabled on my account
according to their end, but I put in a ticket with the request information
to figure out why the request is getting a 403 response.

So to summarize, I'm unsure if this is a PayPal or plugin issue, but if
it's the latter, then the report should be made here, no?
